### ISSUE TITLE
Add a separator for omitted authors within a list of authors

### ISSUE THREAD

**Issue by mjthoraval** [github-young/zotero-better-authors#27]
As discussed in a [previous issue](https://github.com/github-young/zotero-better-authors/issues/16), I would like to be able to identify quickly if some authors have been omitted within a list of authors.

For example, let's uses these settings, showing only the last name of the first and last author:
![image](https://github.com/github-young/zotero-better-authors/assets/36054098/c8ed6f8d-fa34-4a33-af49-920bbbe10c29)

If I have two items with the following authors list:
1. `Smith1 / SmithLast`
2. `Smith1 / Smith2 / SmithLast`

In the current configuration, both of them will be displayed as `Smith1; SmithLast`:
1. `Smith1 / SmithLast` -> `Smith1; SmithLast`
2. `Smith1 / Smith2 / SmithLast` -> `Smith1; SmithLast`

I would like to be able to distinguish these two items. For example with:
1. `Smith1 / SmithLast` -> `Smith1; SmithLast`
2. `Smith1 / Smith2 / SmithLast` -> `Smith1; …SmithLast`

This is still fairly compact and it adds useful information on the authors list.

I think that this was already implemented by @jstzwj in his [PRs](https://github.com/github-young/zotero-better-authors/commits?author=jstzwj):
```
        const lastAuthorDisplayed: string = this.displayAuthorName(authors, authors.length - 1, sep);
        if (firstN < authors.length - 1) {
          return authorList.join(", ") + ", ..., " + lastAuthorDisplayed;
        } else if (firstN == authors.length - 1) {
          return authorList.join(", ") + ", " + lastAuthorDisplayed;
        } else {
          return authorList.join(", ");
        }
```
https://github.com/github-young/zotero-better-authors/blob/185890815740a539b6fb2dd90da6ac702a6b04db/src/modules/lastauthor.ts

But it slowly got rremoved over the iterations.

**Comment by github-young** [comment]
closed in [v3.4.0](https://github.com/github-young/zotero-better-authors/releases/tag/v3.4.0)

**Comment by mjthoraval** [comment]
It seems to work if you display at least one middle author:
![image](https://github.com/github-young/zotero-better-authors/assets/36054098/99111be9-3328-44d0-810b-7b23fbc895d2)

But not if you show only the first and last authos:
![image](https://github.com/github-young/zotero-better-authors/assets/36054098/d3e24c61-949f-4d4e-9d64-6c816425a6b5)

Not to be too greedy, but a user defined separator on this would be nice also :).

**Comment by github-young** [comment]
> But not if you show only the first and last authors

Is this not correct?

> Not to be too greedy, but a user defined separator on this would be nice also :).

OK noted.

**Comment by mjthoraval** [comment]
> Do you mean you want a special separator and for only two authors case?

No.

The first example framed in the second situation has 6 authors.
It is currently displaying:
`Abate, A. R.; Weitz, D. A.`
It should be:
`Abate, A. R.; ...; Weitz, D. A.`
to reflect that some authors have been omitted.

The second example framed is correct.

**Comment by github-young** [comment]
fixed in [v3.5.1](https://github.com/github-young/zotero-better-authors/releases/tag/v3.5.1) and closed.

During implementing this issue, I just realized that I don't have to separate the `first author` and `middle authors` as the former can be merged into the latter. The original code from PR #9 #13 already did this but was changed by me. I just restored it.
